### PR TITLE
Add activity text translations for all remaining bundles

### PR DESCRIPTION
**Pull request by niklasnatter**

### CODE REVIEW

File: src/Sulu/Bundle/CustomUrlBundle/Resources/translations/admin.de.json [hunk 1]
```diff
@@ -14,5 +14,9 @@
     "sulu_custom_url.no_follow": "No Follow",
     "sulu_custom_url.no_follow_info": "Die in der Seite enthaltenen Links werden von der Suchmaschine nicht verfolgt",
     "sulu_custom_url.title_used_for_other_url": "Der Titel \"%title%\" wird bereits für eine andere benutzerdefinierte URL verwendet.",
-    "sulu_custom_url.target_title": "Zielseite"
+    "sulu_custom_url.target_title": "Zielseite",
+    "sulu_activity.text.custom_urls.created": "{userFullName} hat die benutzerdefinierte URL \"{resourceTitle}\" erstellt",
+    "sulu_activity.text.custom_urls.modified": "{userFullName} hat die benutzerdefinierte URL \"{resourceTitle}\" geändert",
+    "sulu_activity.text.custom_urls.removed": "{userFullName} hat die benutzerdefinierte URL \"{resourceTitle}\" gelöscht",
+    "sulu_activity.text.custom_urls.route_removed": "{userFullName} hat eine Route von der die benutzerdefinierten URL \"{resourceTitle}\" gelöscht"
```
Comment (luca-rath) [inline src/Sulu/Bundle/CustomUrlBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_activity.text.custom_urls.route_removed": "{userFullName} hat eine Route von der benutzerdefinierten URL \"{resourceTitle}\" gelöscht"
```

File: src/Sulu/Bundle/ContactBundle/Resources/translations/admin.en.json [hunk 2]
```diff
@@ -72,5 +72,23 @@
...
+    "sulu_activity.text.accounts.contact_added": "{userFullName} has added the contact \"{context_contactName}\" to the organization \"{resourceTitle}\"",
+    "sulu_activity.text.accounts.contact_removed": "{userFullName} has removed the contact \"{context_contactName}\" from the organization \"{resourceTitle}\"",
+    "sulu_activity.text.contact_positions.created": "{userFullName} has created the contact position \"{resourceTitle}\"",
+    "sulu_activity.text.contact_positions.modified": "{userFullName} has created the contact position \"{resourceTitle}\"",
+    "sulu_activity.text.contact_positions.removed": "{userFullName} has created the contact position \"{resourceTitle}\"",
+    "sulu_activity.text.contact_titles.created": "{userFullName} has created the contact title \"{resourceTitle}\"",
+    "sulu_activity.text.contact_titles.modified": "{userFullName} has created the contact title \"{resourceTitle}\"",
+    "sulu_activity.text.contact_titles.removed": "{userFullName} has created the contact title \"{resourceTitle}\""
```
Comment (luca-rath) [inline src/Sulu/Bundle/ContactBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.contact_positions.created": "{userFullName} has created the contact position \"{resourceTitle}\"",
    "sulu_activity.text.contact_positions.modified": "{userFullName} has changed the contact position \"{resourceTitle}\"",
    "sulu_activity.text.contact_positions.removed": "{userFullName} has removed the contact position \"{resourceTitle}\"",
    "sulu_activity.text.contact_titles.created": "{userFullName} has created the contact title \"{resourceTitle}\"",
    "sulu_activity.text.contact_titles.modified": "{userFullName} has changed the contact title \"{resourceTitle}\"",
    "sulu_activity.text.contact_titles.removed": "{userFullName} has removed the contact title \"{resourceTitle}\""
```

File: src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json [hunk 3]
```diff
@@ -71,5 +71,21 @@
...
+    "sulu_activity.text.media.removed": "{userFullName} hat das Medium \"{resourceTitle}\" gelöscht",
+    "sulu_activity.text.media.translation_added": "{userFullName} hat eine Übersetzung für \"{resourceLocale}\" zum Medium \"{resourceTitle}\" hinzugefügt",
+    "sulu_activity.text.media.crop_modified": "{userFullName} hat den Ausschnitt für das Format \"{context_formatKey}\" des Mediums \"{resourceTitle}\" geändert",
+    "sulu_activity.text.media.crop_removed": "{userFullName} hat den Ausschnitt für das Format \"{context_formatKey}\" des Mediums \"{resourceTitle}\" gelöscht",
+    "sulu_activity.text.media.preview_image_modified": "{userFullName} hat das Vorschaubild des Mediums \"{resourceTitle}\" geändert",
+    "sulu_activity.text.media.preview_image_removed": "{userFullName} hat das Vorschaubild des Mediums \"{resourceTitle}\" gelöscht",
+    "sulu_activity.text.media.version_added": "{userFullName} hat eine neue Version für das Medium \"{resourceTitle}\" hinzugefügt",
+    "sulu_activity.text.media.version_removed": "{userFullName} hat eine Version des Mediums \"{resourceTitle}\" gelöscht"
```
Comment (luca-rath) [inline src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json]: It should be possible to show the version using `{context_version}` ... Should we do that?

File: src/Sulu/Bundle/SecurityBundle/Resources/translations/admin.en.json [hunk 4]
```diff
@@ -24,5 +24,15 @@
...
+    "sulu_activity.text.roles.created": "{userFullName} has created the role \"{resourceTitle}\"",
+    "sulu_activity.text.roles.modified": "{userFullName} has modified the role \"{resourceTitle}\"",
+    "sulu_activity.text.roles.removed": "{userFullName} has removed the role \"{resourceTitle}\"",
+    "sulu_activity.text.users.created": "{userFullName} has created the user \"{resourceTitle}\"",
+    "sulu_activity.text.users.modified": "{userFullName} has modified the user \"{resourceTitle}\"",
+    "sulu_activity.text.users.removed": "{userFullName} has removed the user \"{resourceTitle}\"",
+    "sulu_activity.text.users.enabled": "{userFullName} has enabled the user \"{resourceTitle}\"",
+    "sulu_activity.text.users.locked": "{userFullName} has locked the user \"{resourceTitle}\"",
+    "sulu_activity.text.users.unlocked": "{userFullName} has unlocked the user \"{resourceTitle}\"",
+    "sulu_activity.text.users.password_resetted": "{userFullName} has resetted the password of the the user \"{resourceTitle}\""
```
Comment (luca-rath) [inline src/Sulu/Bundle/SecurityBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.users.password_resetted": "{userFullName} has resetted the password of the user \"{resourceTitle}\""
```

File: src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.de.json [hunk 5]
```diff
@@ -8,5 +8,14 @@
...
+    "sulu_activity.text.snippets.copied": "{userFullName} hat das Schnipsel \"{resourceTitle}\" kopiert",
+    "sulu_activity.text.snippets.removed": "{userFullName} hat das Schnipsel \"{resourceTitle}\" gelöscht",
+    "sulu_activity.text.snippets.translation_added": "{userFullName} hat eine neue Sprachvariante für \"{resourceLocale}\" zum Schnipsel \"{resourceTitle}\" hinzugefügt",
+    "sulu_activity.text.snippets.translation_copied": "{userFullName} hat die Sprachvariante für \"{context_sourceLocale}\" des Schnipsels \"{resourceTitle}\" nach \"{resourceLocale}\" kopiert",
+    "sulu_activity.text.snippets.translation_removed": "{userFullName} hat die Sprachvariante für \"{resourceLocale}\" des Schnipsels \"{resourceTitle}\" gelöscht",
+    "sulu_activity.text.webspaces.default_snippet_modified": "{userFullName} has das Standardschnipsel für den Bereich \"{context_snippetAreaKey}\" des Webspaces \"{resourceTitle}\" geändert",
+    "sulu_activity.text.webspaces.default_snippet_removed": "{userFullName} has das Standardschnipsel für den Bereich \"{context_snippetAreaKey}\" des Webspaces \"{resourceTitle}\" gelöscht"
```
Comment (luca-rath) [inline src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_activity.text.webspaces.default_snippet_modified": "{userFullName} has das Standardschnipsel für den Bereich \"{context_snippetAreaKey}\" des Webspace \"{resourceTitle}\" geändert",
    "sulu_activity.text.webspaces.default_snippet_removed": "{userFullName} has das Standardschnipsel für den Bereich \"{context_snippetAreaKey}\" des Webspace \"{resourceTitle}\" gelöscht"
```

I'm not sure about this change

File: src/Sulu/Bundle/TagBundle/Resources/translations/admin.de.json [hunk 6]
```diff
@@ -1,5 +1,9 @@
 {
     "sulu_tag.tags": "Tags",
     "sulu_tag.tag": "Tag",
-    "sulu_tag.no_commas_allowed": "In diesem Feld sind keine Beistriche erlaubt"
+    "sulu_tag.no_commas_allowed": "In diesem Feld sind keine Beistriche erlaubt",
+    "sulu_activity.text.tags.created": "{userFullName} hat den Tag \"{resourceTitle}\" erstellt",
+    "sulu_activity.text.tags.modified": "{userFullName} hat den Tag \"{resourceTitle}\" geändert",
+    "sulu_activity.text.tags.removed": "{userFullName} hat den Tag \"{resourceTitle}\" gelöscht",
+    "sulu_activity.text.tags.merged": "{userFullName} hat den Tag \"{context_sourceTagName}\" in den Tag \"{resourceTitle}\" vereinigt"
```
Comment (luca-rath) [inline src/Sulu/Bundle/TagBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_activity.text.tags.merged": "{userFullName} hat die Tags \"{context_sourceTagName}\" und \"{resourceTitle}\" zusammengeführt"
```

I think `zusammengeführt` sounds a little bit better 🙈

File: src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.de.json (hunk 5 above)
Comment (Prokyonn) [inline src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.de.json]: Maybe we could also change the sentence more like 

```suggestion
    "sulu_activity.text.webspaces.default_snippet_modified": "{userFullName} hat das Standardschnipsel für den Bereich \"{context_snippetAreaKey}\" im Webspace \"{resourceTitle}\" geändert",
    "sulu_activity.text.webspaces.default_snippet_removed": "{userFullName} hat das Standardschnipsel für den Bereich \"{context_snippetAreaKey}\" im Webspace \"{resourceTitle}\" gelöscht"
```

File: src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.en.json [hunk 7]
```diff
@@ -8,5 +8,14 @@
...
+    "sulu_snippet.webspace_default_snippet": "%webspaceKey% default snippet",
+    "sulu_activity.text.snippets.created": "{userFullName} has created the snippet \"{resourceTitle}\"",
+    "sulu_activity.text.snippets.modified": "{userFullName} has modified the snippet \"{resourceTitle}\"",
+    "sulu_activity.text.snippets.copied": "{userFullName} has copied the snippet \"{resourceTitle}\"",
+    "sulu_activity.text.snippets.removed": "{userFullName} has removed the snippet \"{resourceTitle}\"",
+    "sulu_activity.text.snippets.translation_added": "{userFullName} has added a new translation \"{resourceLocale}\" to the snippet \"{resourceTitle}\"",
+    "sulu_activity.text.snippets.translation_copied": "{userFullName} has copied the \"{context_sourceLocale}\" translation of the snippet \"{resourceTitle}\" into \"{resourceLocale}\"",
+    "sulu_activity.text.snippets.translation_removed": "{userFullName} has removed the \"{resourceLocale}\" translation of the snippet \"{resourceTitle}\"",
+    "sulu_activity.text.webspaces.default_snippet_modified": "{userFullName} has modified the default snippet of the area \"{context_snippetAreaKey}\" of the webspace \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.webspaces.default_snippet_modified": "{userFullName} has modified the default snippet for the area \"{context_snippetAreaKey}\" of the webspace \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.de.json (hunk 5 above)
Comment (luca-rath) [reply src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.de.json]: would prefer https://github.com/sulu/sulu/pull/6015#discussion_r629892709

File: src/Sulu/Bundle/ContactBundle/Resources/translations/admin.en.json (hunk 2 above)
Comment (niklasnatter) [reply src/Sulu/Bundle/ContactBundle/Resources/translations/admin.en.json]: 😇

File: src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json (hunk 3 above)
Comment (niklasnatter) [reply src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_activity.text.media.version_removed": "{userFullName} hat die Version {context_version} des Mediums \"{resourceTitle}\" gelöscht"
```

File: src/Sulu/Bundle/ContactBundle/Resources/translations/admin.en.json [hunk 8]
```diff
@@ -72,5 +72,23 @@
     "sulu_contact.delete_media_warning_text": "This operation removes the selected document assignments. The document itself will not be deleted. Do you really wish to proceed?",
     "sulu_contact.add_contact_to_organization": "Add contact to organization",
     "sulu_contact.latitude": "Latitude",
-    "sulu_contact.longitude": "Longitude"
+    "sulu_contact.longitude": "Longitude",
+    "sulu_activity.text.contacts.created": "{userFullName} has created the contact \"{resourceTitle}\"",
+    "sulu_activity.text.contacts.modified": "{userFullName} has modified the contact \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/ContactBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.contacts.modified": "{userFullName} has changed the contact \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/ContactBundle/Resources/translations/admin.en.json [hunk 9]
```diff
@@ -72,5 +72,23 @@
...
+    "sulu_activity.text.contacts.removed": "{userFullName} has removed the contact \"{resourceTitle}\"",
+    "sulu_activity.text.contacts.media_added": "{userFullName} has added the media \"{context_mediaTitle}\" to the contact \"{resourceTitle}\"",
+    "sulu_activity.text.contacts.media_removed": "{userFullName} has removed the media \"{context_mediaTitle}\" from the contact \"{resourceTitle}\"",
+    "sulu_activity.text.accounts.created": "{userFullName} has created the organization \"{resourceTitle}\"",
+    "sulu_activity.text.accounts.modified": "{userFullName} has modified the organization \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/ContactBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.accounts.modified": "{userFullName} has changed the organization \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json [hunk 10]
```diff
@@ -71,5 +71,21 @@
     "sulu_media.form_type": "Form type",
     "sulu_media.circle": "Circle",
     "sulu_media.point": "Point",
-    "sulu_media.rectangle": "Rectangle"
+    "sulu_media.rectangle": "Rectangle",
+    "sulu_activity.text.collections.created": "{userFullName} has created the collection \"{resourceTitle}\"",
+    "sulu_activity.text.collections.modified": "{userFullName} has modified the collection \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.collections.modified": "{userFullName} has changed the collection \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json [hunk 11]
```diff
@@ -71,5 +71,21 @@
...
+    "sulu_activity.text.collections.moved": "{userFullName} has moved the collection \"{resourceTitle}\"",
+    "sulu_activity.text.collections.removed": "{userFullName} has removed the collection \"{resourceTitle}\"",
+    "sulu_activity.text.collections.translation_added": "{userFullName} has added a translation \"{resourceLocale}\" to the collection \"{resourceTitle}\"",
+    "sulu_activity.text.media.created": "{userFullName} has created the media \"{resourceTitle}\"",
+    "sulu_activity.text.media.modified": "{userFullName} has modified the media \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.media.modified": "{userFullName} has changed the media \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json [hunk 12]
```diff
@@ -71,5 +71,21 @@
...
+    "sulu_activity.text.media.moved": "{userFullName} has moved the media \"{resourceTitle}\"",
+    "sulu_activity.text.media.removed": "{userFullName} has removed the media \"{resourceTitle}\"",
+    "sulu_activity.text.media.translation_added": "{userFullName} has added a translation \"{resourceLocale}\" to the media \"{resourceTitle}\"",
+    "sulu_activity.text.media.crop_modified": "{userFullName} has modified the crop for the \"{context_formatKey}\" format of the media \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.media.crop_modified": "{userFullName} has changed the crop for the \"{context_formatKey}\" format of the media \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json [hunk 13]
```diff
@@ -71,5 +71,21 @@
...
+    "sulu_activity.text.media.crop_removed": "{userFullName} has removed the crop for the \"{context_formatKey}\" format from the media \"{resourceTitle}\"",
+    "sulu_activity.text.media.preview_image_modified": "{userFullName} has modified the preview image of the media \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.media.preview_image_modified": "{userFullName} has changed the preview image of the media \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/SecurityBundle/Resources/translations/admin.en.json [hunk 14]
```diff
@@ -24,5 +24,15 @@
     "sulu_security.inherit_permissions_title": "Inherit permissions?",
     "sulu_security.inherit_permissions_label": "Override permissions of sub elements",
     "sulu_security.move_permission_title": "Notice",
-    "sulu_security.move_permission_warning": "Permissions from the parent element will not be transferred."
+    "sulu_security.move_permission_warning": "Permissions from the parent element will not be transferred.",
+    "sulu_activity.text.roles.created": "{userFullName} has created the role \"{resourceTitle}\"",
+    "sulu_activity.text.roles.modified": "{userFullName} has modified the role \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/SecurityBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.roles.modified": "{userFullName} has changed the role \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/SecurityBundle/Resources/translations/admin.en.json [hunk 15]
```diff
@@ -24,5 +24,15 @@
...
+    "sulu_activity.text.roles.removed": "{userFullName} has removed the role \"{resourceTitle}\"",
+    "sulu_activity.text.users.created": "{userFullName} has created the user \"{resourceTitle}\"",
+    "sulu_activity.text.users.modified": "{userFullName} has modified the user \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/SecurityBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.users.modified": "{userFullName} has changed the user \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.en.json [hunk 16]
```diff
@@ -8,5 +8,14 @@
     "sulu_snippet.default_snippets": "Default Snippets",
     "sulu_snippet.snippet_area": "Type or area",
     "sulu_snippet.taxonomies": "Taxonomies",
-    "sulu_snippet.webspace_default_snippet": "%webspaceKey% default snippet"
+    "sulu_snippet.webspace_default_snippet": "%webspaceKey% default snippet",
+    "sulu_activity.text.snippets.created": "{userFullName} has created the snippet \"{resourceTitle}\"",
+    "sulu_activity.text.snippets.modified": "{userFullName} has modified the snippet \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.snippets.modified": "{userFullName} has changed the snippet \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/TagBundle/Resources/translations/admin.en.json [hunk 17]
```diff
@@ -1,5 +1,9 @@
 {
     "sulu_tag.tags": "Tags",
     "sulu_tag.tag": "Tag",
-    "sulu_tag.no_commas_allowed": "There are no commas allowed in this field"
+    "sulu_tag.no_commas_allowed": "There are no commas allowed in this field",
+    "sulu_activity.text.tags.created": "{userFullName} has created the tag \"{resourceTitle}\"",
+    "sulu_activity.text.tags.modified": "{userFullName} has modified the tag \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/TagBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.tags.modified": "{userFullName} has changed the tag \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/WebsiteBundle/Resources/translations/admin.en.json [hunk 18]
```diff
@@ -17,5 +17,8 @@
     "sulu_website.body_close": "Body close",
     "sulu_website.cache_clear": "Clear website cache",
     "sulu_website.cache_clear_warning_title": "Clear website cache?",
-    "sulu_website.cache_clear_warning_text": "This operation will clear the entire cache for all webspaces. Do you really wish to proceed?"
+    "sulu_website.cache_clear_warning_text": "This operation will clear the entire cache for all webspaces. Do you really wish to proceed?",
+    "sulu_activity.text.analytics.created": "{userFullName} has created the analytics item \"{resourceTitle}\"",
+    "sulu_activity.text.analytics.modified": "{userFullName} has modified the analytics item \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/WebsiteBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.analytics.modified": "{userFullName} has changed the analytics item \"{resourceTitle}\"",
```

File: src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.en.json [hunk 19]
```diff
@@ -8,5 +8,14 @@
...
+    "sulu_activity.text.snippets.copied": "{userFullName} has copied the snippet \"{resourceTitle}\"",
+    "sulu_activity.text.snippets.removed": "{userFullName} has removed the snippet \"{resourceTitle}\"",
+    "sulu_activity.text.snippets.translation_added": "{userFullName} has added a new translation \"{resourceLocale}\" to the snippet \"{resourceTitle}\"",
+    "sulu_activity.text.snippets.translation_copied": "{userFullName} has copied the \"{context_sourceLocale}\" translation of the snippet \"{resourceTitle}\" into \"{resourceLocale}\"",
+    "sulu_activity.text.snippets.translation_removed": "{userFullName} has removed the \"{resourceLocale}\" translation of the snippet \"{resourceTitle}\"",
+    "sulu_activity.text.webspaces.default_snippet_modified": "{userFullName} has modified the default snippet for the area \"{context_snippetAreaKey}\" of the webspace \"{resourceTitle}\"",
```
Comment (Prokyonn) [inline src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_activity.text.webspaces.default_snippet_modified": "{userFullName} has changed the default snippet for the area \"{context_snippetAreaKey}\" of the webspace \"{resourceTitle}\"",
```

**Review by Prokyonn — APPROVED** [review]

**Review by luca-rath — APPROVED** [review]

**Pull request merged**